src/pages/commitqueue/CommitQueueCard.tsx

```diff
@@ -78,6 +95,7 @@ export const CommitQueueCard: React.FC<Props> = ({
             </Container>
           </CommitInfo>
         ) : (
+          // should only get here for pull requests not processed yet (ie. added in the past minute)
           <CommitInfo>
             <PRCardTitle
               href={`https://github.com/${owner}/${repo}/pull/${issue}`}
```

Comment:
Can we use the util function you created.

Reply:
oops fixed